orlando_recycling/settings.py

```diff
@@ -28,7 +28,7 @@
 SECRET_KEY = get_env("DJANGO_SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True if get_env("DJANGO_DEBUG") == "True" else False
```

Comment:
This needs to be reverted back to the way it was. "DEBUG" needs to remain an environment variable so it can be optional during the deployment of the dev server, etc.